users/models.py

```diff
@@ -1,4 +1,14 @@
 from django.contrib.auth.models import AbstractUser
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
 
 class User(AbstractUser):
     pass
+
+
+@receiver(post_save, sender=User)
+def create_auth_token(sender, instance=None, created=False, **kwargs):
```

Comment:
`signals` should be avoided if possible ;) they're hard to debug, and not always firing :( you could just create the token in the  `User.save` method, it'll be more reliable